fiware-platform/values.yaml

```diff
@@ -80,7 +80,7 @@ applications:
     - values.yaml
 
   - name: airquality-simulator
-    enabled: true
+    enabled: false
```

Comment:
Just a reminder to turn this to true again before merging into main

Reply:
Good point! We forgot to tell you, setting it to false does not disable it